.travis.yml

```diff
@@ -5,9 +5,13 @@ env:
 - TOX_ENV=py27
 - TOX_ENV=py33
 - TOX_ENV=py34
+- TOX_ENV=py27-build-theme-mkdocs
+- TOX_ENV=py34-build-theme-mkdocs
+- TOX_ENV=py27-build-theme-readthedocs
+- TOX_ENV=py34-build-theme-readthedocs
```

Comment:
Does the tipue search change mean we want these in particularly, or is that just an accidental change?